Extension/src/LanguageServer/client.ts

```diff
@@ -1842,7 +1842,8 @@ export class DefaultClient implements Client {
     public onDidChangeTextDocument(textDocumentChangeEvent: vscode.TextDocumentChangeEvent): void {
         if (util.isCpp(textDocumentChangeEvent.document)) {
             // If any file has changed, we need to abort the current rename operation
-            if (workspaceReferences.renamePending) {
+            if (workspaceReferences !== undefined // Occurs when a document changes before cpptools starts.
+                && workspaceReferences.renamePending) {
```

Comment:
Should I use `?.` instead?

Reply:
I think it's better with an explicit checks. Otherwise, I'd have to make the comment longer.